lib/99 bottles.rb

```diff
@@ -0,0 +1,17 @@
+  def ninety_nine_bottles_of_beer
+    bottles = 100
+    bottles.downto(2) do |bottles|
+      if (bottles-1) > 1
+      puts (bottles-1).to_s + " Bottles of beer on the wall, " +
+      (bottles-1).to_s + " Bottles of beer! You take one down, you pass it
+      around and " + (bottles-2).to_s + " Bottles of beer on the wall!"
+      else
+        puts (bottles-1).to_s + " Bottle of beer on the wall, " +
+      (bottles-1).to_s + " Bottle of beer! You take one down, you pass it
+      around and " + (bottles-2).to_s + " Bottle of beer on the wall!"
+    end
+    end
+    # bottles = (bottles-1)
+  end
+
+puts  ninety_nine_bottles_of_beer
```

Comment:
Unnecessary spacing detected.<br>Inconsistent indentation detected.<br>Put one space between the method name and the first argument.<br>Do not write to stdout. Use Rails' logger if you want to log.